glossary.md

```diff
@@ -145,6 +228,10 @@ The mechanism of Region split is to use one initial Region to cover the entire k
 
 Restore is the reverse of the backup operation. It is the process of bringing back the system to an earlier state by retrieving data from a prepared backup.
 
+### RocksDB
+
+RocksDB is an LSM-tree structured engine that provides key-value storage and read-write functionality. It was developed by Facebook based on LevelDB. RocksDB is the core storage engine of TiKV, used for storing Raft logs and user data.
```

Comment:
```suggestion
RocksDB is an LSM-tree structured engine that provides key-value storage and read-write functionality. It was developed by Facebook and is based on LevelDB. RocksDB is the core storage engine of TiKV.
```

Storing Raft logs can also be done in "Raft Engine" isn't it?

Maybe link to https://rocksdb.org/ ?